src/hacbs/components/ApplicationDetails/ApplicationQuickstartContent.ts

```diff
@@ -0,0 +1,50 @@
+export const applicationQuickstartContent = {
+  apiVersion: 'console.openshift.io/v1',
+  kind: 'QuickStarts',
+  metadata: {
+    name: 'hacbs-getting-started-app',
+  },
+  spec: {
+    version: 4.7,
+    displayName: `Getting started with an app`,
+    durationMinutes: 60,
+    description: `Getting started with an app.`,
+    introduction: `You created an app and added a component to it. Good job! To get started using your app, follow the steps in this quick start guide.`,
+    tasks: [
+      {
+        title: `Add components and builds`,
+        description: `
+1. To add components and builds, go to the **Actions** dropdown and select "Add Component".
+2. Add the components you want included, then you will receive a PR to add the build pipeline to your code.
+3. View the Pull Requests we have sent you by clicking on the **Merge Pull Requests** button. You will be directed to the Pull Requests list.
+4. Customize this build pipeline according to your development needs.
+5. Merge to the PR to add the build pipeline to your code.`,
+      },
+      {
+        title: `Add integration test`,
+        description: `Integration test is a pipeline, set from GitHub, with the goal to test each component individually and then once more together as one application. Because it is unique per app, this task is on you. But we can help you get there. Check out our guide on how to set it up!  
+            \
+            [Learn about integration tests](http://www.redhat.com)
```

Comment:
Comment out link until we have a proper url. Add a `TODO`.

Reply:
i did add todos but it makes sense to keep these as nonfunctioning links for content purposes, i changed them to do nothing instead of go to redhat.com